.travis.yml

```
@@ -40,5 +40,12 @@ jobs:
         - npx aegir test -t browser -- --browsers FirefoxHeadless
         - npx aegir test -t webworker -- --browsers FirefoxHeadless
 
+    - stage: test
+      name: interop
+      script:
+        - cd node_modules/interop-libp2p
+        - npm install
+        - LIBP2P_JS=./../../../../../src/index.js npx aegir test -t node --bail
```

I think we should be able to use the travis build directory to simplify this.
```suggestion
        - LIBP2P_JS=${TRAVIS_BUILD_DIR}/src/index.js npx aegir test -t node --bail
```